MetaMask/metamask-extension
# Snaps devx fix

Fixes: #13011 

Explanation:  This fixes a devx issue when running a build-type of 'flask' in dev mode. On reload, the build process would break if there were updates made to a file that had code fences. The reason was that the `removeFencedCode` transform stream was being called on twice, the first time it works fine and transforms properly, however the second time is done on a transpiled version of the same file and the code fencing regex does not sit well with this hence the various thrown errors. Regex was added to check for if file content had a source map inclusion and the function execution is halted at this point.

Manual testing steps:  
  -  Run `yarn start --build-type flask`
  -  Make changes in a file with code fencing and observe that the build process reloads properly.
